libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1023,6 +1027,15 @@ void SMTEncoder::visitPublicGetter(FunctionCall const& _funCall)
 	}
 }
 
+bool SMTEncoder::shouldAnalyze(ContractDefinition const& _contract) const
+{
+	if (!_contract.canBeDeployed())
+		return false;
+
+	return m_settings.contracts.all() ||
+		m_settings.contracts.has(_contract.sourceUnitName(), _contract.name());
```

Comment:
```suggestion
	return 
		m_settings.contracts.all() ||
		m_settings.contracts.has(_contract.sourceUnitName(), _contract.name());
```

Reply:
I would prefer the current way if you don't mind too much